packages/xod-client/src/project/utils.js

```diff
@@ -25,24 +25,8 @@ export const isLinkConnectedToNodeIds = R.uncurryN(2, nodeIds =>
 // pins linking validation utils
 //
 
-// :: RenderablePin -> String | Null
-export const getPinSelectionError = pin => {
-  if (XP.isInputPin(pin) && pin.isConnected) {
-    return LINK_ERRORS.ONE_LINK_FOR_INPUT_PIN;
-  }
-
-  return null;
-};
-
-// :: RenderablePin -> Boolean
-export const isPinLinkable = R.complement(getPinSelectionError);
-
 // :: RenderablePin -> RenderablePin -> String | Null
 export const getLinkingError = R.curry((pin1, pin2) => {
-  if (!isPinLinkable(pin1) || !isPinLinkable(pin2)) {
-    return LINK_ERRORS.ONE_LINK_FOR_INPUT_PIN;
```

Comment:
Looks like the `ONE_LINK_FOR_INPUT_PIN` is no longer required thus should be removed:

```bash
$ rg ONE_LINK_FOR_INPUT_PIN
packages/xod-client/src/editor/constants.js:  ONE_LINK_FOR_INPUT_PIN: 'ONE_LINK_FOR_INPUT_PIN',
packages/xod-client/src/project/messages.js:  [LE.ONE_LINK_FOR_INPUT_PIN]: composeMessage(
```